### ISSUE TITLE
Geometry column While saving to db

### ISSUE THREAD

**Issue by jeyasurya520** [robertjkeller/PyARINC424#4]
Is ther any way to add geometry column on creating table in db for each layer like (airport,waypoint,FIR,VOR.. etc) ?

**Comment by robertjkeller** [comment]
No, that's sort of outside the scope of this tool. You would need additional tooling, or maybe you could set up stored SQL procedures to add those columns.

**Comment by jeyasurya520** [comment]
> No, that's sort of outside the scope of this tool. You would need additional tooling, or maybe you could set up stored SQL procedures to add those columns.

Thanks for the replay @robertjkeller . And is that any function to get FIR.json from controlled_airspace table ,i need a help in that.

**Comment by bitwisetech** [comment]
Many thanks for this package, its clarity has helped someone with no prior experience of database interaction.
